content/support/pricing.md

```diff
@@ -15,27 +15,31 @@ Our Centrally-Organised Workshops bring hands-on computational and data skill wo
 
 
 {{< table >}}
-|                    |Workshop Fee (USD)|
-|------------------- |---------------   |
-|High Income         |$3,000            |
-|Upper-Middle Income |$2,250            |
-|Lower-Middle Income |$1,500            |
-|Low Income          |$750              |
+| Income Level         | Workshop Fee (USD) | Membership fee |
```

Comment:
I am concerned about the added complexity to the tables of showing both the a la carte fee and membership fee, especially given the other addition of corporate pricing (see https://github.com/carpentries/carpentries.org/pull/486). This table would now need to have four price columns (a la carte fee (non-profit), member fee (non-profit), a la carte fee (corporate), member fee (corporate)). For simplicity, I would prefer to have a footnote to each table noting that Members are eligible for a 40% discount off the listed fees.

Reply:
@ErinBecker Yes that makes more sence - can we first merge https://github.com/carpentries/carpentries.org/pull/486, then I can add the sentence?